_includes/docs/devices-library/blocks/microcontrollers/camera-code-to-program-block.md

```diff
@@ -24,7 +24,7 @@ To do this you can change the value of variable **DEMO_MODE** to **1**:
 #include "soc/soc.h"
 #include "soc/rtc_cntl_reg.h"
 
-#define THINGSBOARD_ENABLE_DYNAMIC 1
+#define DEMO_MODE 0
```

Comment:
What is this about?

Reply:
We need DEMO_MODE here to reduce the quality of the photo, to avoid exceeding message size on demo.thingsboard.io.